### ISSUE TITLE
Color scheme customization

### ISSUE THREAD

**Issue by nazaradeh** [nick-delirium/lemmy-fennec#11]
It would be nice to be able to customize the colour scheme of the app. It might also help with the app's vision for it being accessible.

I'm sure the current color scheme is well thought out and already fits well with most vision impairments, but my only issue is the dark blue icons (e.g. upvote, downvote, share link...). It's just a little difficult to look at for me personally.

I also understand if you want the app to be more minimal with fewer customization options. But this is a feature that I would quite like, personally.

**Comment by nick-delirium** [comment]
I agree with dark blue being not the best color in dark scheme and its actually gonna be something like [#637ac4](https://www.color-hex.com/color/637ac4) in next version :)

When I'll have some time to split settings from user store, I'll make entire palette customisable with an option to return to default with a single tap probably
![image](https://github.com/nick-delirium/lemmy-fennec/assets/23126999/665acddd-f8e6-4754-a2bb-0a6fa9be6349)

**Comment by nick-delirium** [comment]
[Partially fixed](https://github.com/nick-delirium/lemmy-fennec/releases/tag/0.0.1-alpha.2) but I'm keeping this issue to work on custom theme setting.
